src/server/Html.js

```diff
@@ -21,11 +21,17 @@ export default function Html({store, entries, StyleSheetServer, renderProps}) {
   // const initialState = `window.__INITIAL_STATE__ = ${JSON.stringify(store.getState())}`;
   // <script dangerouslySetInnerHTML={{__html: initialState}}/>
   const {html, css} = StyleSheetServer.renderStatic(() => {
-    return renderToString(
-      <Provider store={store}>
-        <RouterContext {...renderProps} />
-      </Provider>
-    );
+    try {
```

Comment:
For whatever reason this exception does not bubble up to Raven without the try/catch block on this function. What should we put here?